.github/workflows/windows-ci.yml

```diff
@@ -57,23 +65,33 @@ jobs:
           msystem: ${{ matrix.msystem }}
       - name: Build project
         run: |
-          [[ ${{ matrix.tests }} == "none" ]] && debug="" suffix="" || debug="--enable-debug" suffix="-dbg"
-          ./coinbrew/coinbrew build ${{ github.event.repository.name }} --no-prompt --skip='ThirdParty/Metis ThirdParty/Mumps' --build=x86_64-w64-mingw32 ${{ env.host_flag }} $debug --verbosity 4 --tests ${{ matrix.tests }} --enable-relocatable
+          [[ ${{ matrix.debug }} == "false" ]] && debug="" tests="none" || debug="--enable-debug" tests="main"
```

Comment:
What we had before was `--tests main` when `debug` is `false` and `--tests none` when `debug` is true, but I think that was only to speed things up on Appveyor and I would be happy for now if we just do the testing in all cases.